### DISCUSSION TITLE
后续还会集成DC-SDK的新版本吗

### DISCUSSION THREAD

**Discussion by toadprince** [zouyaoji/vue-cesium#496] [General]
DC-SDK从3.0开始就把依赖从 Cesium 换成 @cesium/engin，[可以看这里](https://github.com/dvgis/dc-sdk/releases)，当前vue-cesium队DC的集成方式，不适用于3.0+，会报错，请问后续还会集成DC的新版本吗？

**Comment by zouyaoji** [comment]
我看看，好集成就集成。

**Reply by toadprince** [reply]
有可能需要指定自定义cesium，https://resource.dvgis.cn/dc-docs/zh/api/global#ready，DC.ready({Cesium:''})

**Comment by zouyaoji** [comment]
刚尝试了一下，新的dc-sdk不好集成了，新的dc-sdk用的@cesium.engine，精简了些内容，vue-cesium不适用。

要集合vue-cesium+ dc-sdk用的话可以考虑dc-sdk提供的自定义cesium引入方式，再用vc-viewer提供的`viewerCreator`自定义viewer的构造方法来初始化viewer。

**Comment by zouyaoji** [comment]
还是适配了一下，[详见](https://zouyaoji.top/vue-cesium/#/zh-CN/component/platforms/vc-demo-dc-sdk)

**Comment by cavencj** [comment]
@zouyaoji  感谢支持
